src/mkdocs/docs/guides/ssm-quick-setup-patching/ssm-quick-setup-patching-configuration.md

```diff
@@ -0,0 +1,143 @@
+3. Paste the content into a text editor
+4. Search for **RetentionInDays** properties on the 'AWS::Logs::LogGroup' resources. Remove all instances of this property and **Save** the file
+5. Click on the 'AWS-QuickSetup-PatchPolicy-TA-...' StackSet, and click the **StackSet info** tab.
+6. Copy & Paste all the ID's within the **Organziational unit IDs** to a text file, as this will be needed later.
+7. From the **Actions** dropdown, choose **Edit StackSet details**
+8. Click **Replace current template** and follow the steps to upload the modified saved template from step 4.
+9.  Accept all defaults until **Step 2**, and update the following parameters:
+   1.  **OutputS3BucketName** -> The Central Log Archive S3 bucket in the logging account. Example, "asea-logarchive-phase0-cacentral1-lgh04fj6ulma"
+   2.  **OutputS3KeyPrefix** -> Enter "ssm-patching"
+10. Accept all other defaults and progress until **Step 4 - Set deployment options**
+11. The **Organizationl units (OU)** does not maintain the original values. Add each value that was copied in step 6.
+12. Specify the same region(s) used in the original configuration (e.g. ca-central-1)
+13. Accept all other defaults, and deploy the StackSet update
+
+Review the CloudFormation Stack Sets, and wait for the 'AWS-QuickSetup-PatchPolicy-MA-...' StackSet to succeed.
+
+1. Click on the 'AWS-QuickSetup-PatchPolicy-MA-...' StackSet, and from the **Actions** dropdown, choose **Edit StackSet details**
+2. Accept all defaults until **Step 2**, and update the following parameters:
+   1.  **OutputS3BucketName** -> The Central Log Archive S3 bucket in the logging account. Example, "asea-logarchive-phase0-cacentral1-lgh04fj6ulma"
+   2.  **OutputS3KeyPrefix** -> Enter "ssm-patching"
+3.  Accept all other defaults, and deploy the StackSet update
+
+The StackSets should be successfully deployed, and you can return to the Quick Setup Patch Policy to see the dashboard showing Success.
+
+
+### 1.2.3 ASEA Configuration file updates
+
+> This section assumes v1.5.7 or greater as it includes the permission policy, ssm document, and custom config rule. Note that these can be manually uploaded to the config S3 bucket, and thus available to older ASEA versions.
+
+1. Click on the 'AWS-QuickSetup-PatchPolicy-TA-...' StackSet, and click the **Parameters** tab.
+2. Locate and record the value for the **QSConfigurationId**
+
+3. Review the sample configuration file, **config.SSM-Patching-example**, found in **reference-artifacts/SAMPLE_CONFIGS/**
+
+4. Compare the file to the sample, **config.example.json**
+
+5. Note how the SSM Document is registered and refereced in target OUs. Adjust as needed.
+
+6. Note how the new IAM Policy is applied to different Roles. Adjust as necessary.
+
+7. Review and adjust the custom AWS Config Rule. The value record from step 1, is needed. Replace the **"*** REPLACE AFTER QUICK SETUP ***"** with the value. As an example, it may look like **vair8**.
```

Comment:
could say: "It should be a 5 characters alphanumeric string, for example **vair8** or **rfnce**"

Reply:
Thanks for the feedback, I've updated the doc!